BikeRentalApi/jobs/minutely/clear_expired_reservations.py

```diff
@@ -0,0 +1,20 @@
+from django.utils import timezone
+from django_extensions.management.jobs import MinutelyJob
+
+from BikeRentalApi.enums import BikeState
+from BikeRentalApi.models import Reservation
+
+
+class Job(MinutelyJob):
+    help = "Periodic job for clearing expired reservations."
+
+    def execute(self):
+        date_now = timezone.now()
+
+        for res in Reservation.objects.filter(expire_date__lt = date_now):
```

Comment:
`=` is not a comparison.
I would use <= because datetime object are compared down to miliseconds.
```suggestion
        for res in Reservation.objects.filter(expire_date__lt <= date_now):
```